server/usr/libexec/nethserver/postfix-create-DH2048

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2020 Nethesis S.r.l.
+# http://www.nethesis.it - support@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+#
+# nethserver-mail-server
+#
+# Create the diffie hellman 2048 bit of postfix
+#
+# The DH group could be renewed from time to time for security concerns.
+
+/usr/bin/openssl dhparam -out /etc/pki/dovecot/private/postfix.dh.param.tmp 2048 > /dev/null 2>&1
+
+if [[ $? != 0 ]];then
+    exit 1;
```

Comment:
`.tmp` file is not cleaned up. Set a `trap` to ensure it is removed on EXIT (e.g. https://github.com/NethServer/nethserver-base/blob/7c1f0e7e4e111189dcfa0f4edca868a14c32e492/root/etc/e-smith/events/actions/nethserver-certificate-upload#L32)